docs/src/almalinux_instructions.rst

```diff
@@ -0,0 +1,257 @@
+
+.. code-block:: shell
+    
+    # Create a custom MongoDB repo file:
+    sudo cat <<EOT >> /etc/yum.repos.d/mongodb-org.repo
+    [mongodb-org-4.4]
+    name=MongoDB Repository
+    baseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/4.4/x86_64/
+    gpgcheck=1
+    enabled=1
+    gpgkey=https://www.mongodb.org/static/pgp/server-4.4.asc
+    EOT
+
+    # Install MongoDB:
+    sudo dnf update
+    sudo dnf install -y mongodb-org
+    
+    # Start and enable MongoDB to run on startup:
+    sudo systemctl start mongod
+    sudo systemctl enable mongod
+
+    # Check that it is working
+    sudo systemctl status mongod
+
+.. note::
+
+    On RHEL7/CentOS7, MongoDB v14 can be installed as follows:
+    ``curl -fsSL https://rpm.nodesource.com/setup_14.x | bash - && yum install -y nodejs``
+
+
+3. Install carta-controller
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The easiest way to install carta-controller is using ``npm``. 
+
+.. code-block:: shell
+
+    sudo dnf install -y python3 make gcc-c++
+    sudo npm install -g --unsafe-perm carta-controller
+
+.. note::
+
+    The carta-controller executable will be installed at ``/usr/local/lib/node_modules/carta-controller``.
+    The carta-frontend will be installed at ``/usr/local/lib/node_modules/carta-controller/node_modules/carta-frontend/build``.
+
+.. note::
+    
+    Do not pass the ``--unsafe-perm`` flag to ``npm`` if using a local install.
+
+.. note::
+    Beta versions of carta-controller can be installed. For example, ``sudo npm install -g --unsafe-perm carta-controller@beta``. 
+    Available versions can be found `here <https://www.npmjs.com/package/carta-controller>`_.
+
+.. note::
+    
+    On RHEL7/CentOS7 the carta-controller package can not run with the default gcc version 4.8.5 (there would be an error due to ``node-linux-pam``). 
+    A work around is to install a newer GCC version from source in order to get a newer ``libstdc++.so.6``, then add the location of the newer 
+    ``libstdc++.so.6`` to the LD_LIBRARY_PATH. After that, carta-controller can run on RHEL7/CentOS7.
+
+
+4. Install the carta-backend component
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The easiest way may be to install the carta-backend is from our cartavis RPM repository.
+
+.. code-block:: shell
+
+    # Install carta-backend
+    sudo curl https://packages.cartavis.org/cartavis-el8.repo --output /etc/yum.repos.d/cartavis.repo
+    sudo dnf -y install 'dnf-command(config-manager)'
+    sudo dnf -y install epel-release
+    sudo dnf -y config-manager --set-enabled powertools
+    sudo dnf -y install carta-backend
+
+    # Check that the backend can run and matches the major version number of the controller
+    /usr/bin/carta_backend --version
+
+
+.. note::
+    
+    If you install the beta version of carta-controller, you need to install the beta version of the carta-backend ``sudo dnf -y install carta-backend-beta``.
```

Comment:
This entire document should be just for the beta version, and contain all the correct version names -- it should be backported to the release branch with different version names.